lib/streamlit/commands/page_config.py

```diff
@@ -48,6 +56,19 @@ def set_page_config(
         How the sidebar should start out. Defaults to "auto",
         which hides the sidebar on mobile-sized devices, and shows it otherwise.
         "expanded" shows the sidebar initially; "collapsed" hides it.
+    menu_items: dict
+        Configure the menu that appears on the top-right side of this app.
+        The keys in this dict denote the menu item you'd like to configure:
+                - "Get help": str or None
+                        The URL this menu item should point to.
+                        If None, hides this menu item.
+        - "Report a Bug": str or None
+                        The URL this menu item should point to.
+                        If None, hides this menu item.
+                - "About": str or None
+                        A markdown string to show in the About dialog.
+                        If None, only shows Streamlit's default About text.
```

Comment:
nit: the indentation here looks a bit weird I'd expect this to look something like
```python
"""
    menu_items: dict
        Configure the menu that appears on the top-right side of this app.
        The keys in this dict denote the menu item you'd like to configure:
          - "Get help": str or None
            The URL this menu item should point to.
            If None, hides this menu item.
          - "Report a Bug": str or None
            The URL this menu item should point to.
            If None, hides this menu item.
          - "About": str or None
            A markdown string to show in the About dialog.
            If None, only shows Streamlit's default About text.
"""
```